src/Tizen.Applications.Common/Tizen.Applications/ApplicationRunningContext.cs

```diff
@@ -156,6 +159,34 @@ public string ApplicationId
             }
         }
 
+        /// <summary>
+        /// Gets whether the application is terminated.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsTerminated
```

Comment:
This change is for supporting legacy native API feature in C# API. I think changing State property to support live state is a kind of API semantic break. I'd like to keep IsTerminated property seperately.

However, I think it would be better if there is a 'remark' explaining this can be used to check 'current state' of RunningContext in contrast with State property.